### PR TITLE
feat(shipping): style shipping discount

### PR DESCRIPTION
**Pull request by ardiankristo**
Figma:
[Estimate Shipping](https://www.figma.com/file/TDlhTnv6EtCpPHEi0u7xMS/Framic-v.02?node-id=906%3A16395&t=Ng3WUTzqKN9xCmwD-1)
[Shipping Method](https://www.figma.com/file/AMtQECMzDn4EXf4geUmzUI/%F0%9F%9F%A2-%5BVENDOR%5D-Framic?node-id=3962%3A21108&t=oSlY0P3cJjyvEwiw-1)
[Order History](https://www.figma.com/file/AMtQECMzDn4EXf4geUmzUI/%F0%9F%9F%A2-%5BVENDOR%5D-Framic?node-id=4070%3A21422&t=HjVTbwH3sffGKCDD-4)
[Payment Notif](https://www.figma.com/file/AMtQECMzDn4EXf4geUmzUI/%F0%9F%9F%A2-%5BVENDOR%5D-Framic?node-id=4084%3A21424&t=F3rKnEUcv0g1cCiH-4)

Product detail shipping estimation:
Screenshot from 2023-04-25 12-27-16
Screenshot from 2023-04-25 12-27-18

Shipping method & order summary box:
Screenshot from 2023-04-25 12-58-26
Screenshot from 2023-04-25 13-19-41

Payment Notif:
Screenshot from 2023-04-25 13-35-11
Screenshot from 2023-04-25 14-12-46

Order History:
Screenshot from 2023-04-25 13-35-17
Screenshot from 2023-04-25 14-14-40

### CODE REVIEW

**Comment by ardiankristo** [comment]
Add handling for long shipping method text:
Screenshot from 2023-04-25 15-48-52
Screenshot from 2023-04-25 15-51-25
Screenshot from 2023-04-25 15-49-48
Screenshot from 2023-04-25 15-49-52

File: public/scss/pages/ShippingMethod.module.scss [hunk 1]
```diff
@@ -65,7 +65,12 @@
   {
     margin-left: 4px;
     order: 2;
-    max-width: 60%;
+    max-width: 80%;
+
+    @media screen and (max-width: 575px) {
```
Comment (yohanavitrawati) [inline public/scss/pages/ShippingMethod.module.scss:70]: tanda { di-enter ya

File: public/scss/pages/ShippingMethod.module.scss [hunk 2]
```diff
@@ -74,6 +79,10 @@
     color: $color_black_text;
     margin: 0;
     white-space: break-spaces;
+
+    @media screen and (max-width: 575px) {
```
Comment (yohanavitrawati) [inline public/scss/pages/ShippingMethod.module.scss:83]: sama tanda { di-enter ya

**Review by yohanavitrawati — CHANGES_REQUESTED** [review]

**Review by yohanavitrawati — APPROVED** [review]

**Review by iqbalwijonarko — APPROVED** [review]

**Comment by mayun19** [comment]
Wording discount nya uppercase to di figma nya nya to all 
image


> Figma: [Estimate Shipping](https://www.figma.com/file/TDlhTnv6EtCpPHEi0u7xMS/Framic-v.02?node-id=906%3A16395&t=Ng3WUTzqKN9xCmwD-1) [Shipping Method](https://www.figma.com/file/AMtQECMzDn4EXf4geUmzUI/%F0%9F%9F%A2-%5BVENDOR%5D-Framic?node-id=3962%3A21108&t=oSlY0P3cJjyvEwiw-1) [Order History](https://www.figma.com/file/AMtQECMzDn4EXf4geUmzUI/%F0%9F%9F%A2-%5BVENDOR%5D-Framic?node-id=4070%3A21422&t=HjVTbwH3sffGKCDD-4) [Payment Notif](https://www.figma.com/file/AMtQECMzDn4EXf4geUmzUI/%F0%9F%9F%A2-%5BVENDOR%5D-Framic?node-id=4084%3A21424&t=F3rKnEUcv0g1cCiH-4)
> 
> Product detail shipping estimation: Screenshot from 2023-04-25 12-27-16 Screenshot from 2023-04-25 12-27-18
> 
> Shipping method & order summary box: Screenshot from 2023-04-25 12-58-26 Screenshot from 2023-04-25 13-19-41
>

File: public/scss/pages/ShippingMethod.module.scss [hunk 3]
```diff
@@ -50,21 +51,63 @@
...
+    }
+  }
+
+  &_shippingNameDiv
+  {
+    margin-left: 4px;
+    order: 2;
+    max-width: 80%;
+
+    @media screen and (max-width: 575px)
```
Comment (mayun19) [inline public/scss/pages/ShippingMethod.module.scss:70]: use mixin $breakpoint_max_sm

File: public/scss/pages/ShippingMethod.module.scss [hunk 4]
```diff
@@ -50,21 +51,63 @@
...
   }
 
   &_shippingName
   {
     @include typographyBuilder(400, 16, 24);
     color: $color_black_text;
     margin: 0;
+    white-space: break-spaces;
+
+    @media screen and (max-width: 575px)
```
Comment (mayun19) [inline public/scss/pages/ShippingMethod.module.scss:84]: idem

**Review by mayun19 — CHANGES_REQUESTED** [review]

**Comment by ardiankristo** [comment]
Screenshot from 2023-04-28 14-36-16
Screenshot from 2023-04-28 14-37-59

**Review by mayun19 — APPROVED** [review]

**Pull request merged**